dash/frontend/src/app/modules/shared/directives/phone-mask.directive.ts

```diff
@@ -0,0 +1,65 @@
+import { Directive, Input, OnInit, OnDestroy } from '@angular/core';
```

Comment:
This looks like it only supports US phone numbers, I believe the simpler validation we had before was so that we could support international phone numbers without pulling in a "real" phone validation library. May want to double check with Jake/Jason before maknig this change.

Reply:
There was no validation before. I can pull it into a separate PR though.

Reply:
Recheck